From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Tue, 1 Oct 2019 13:00:00 +0200
Subject: [PATCH 1/1] Make libraries private within the pkg-config file

Assuming that IlmBase is built statically, this ensures that the
libraries are not exposed during linking.

diff --git a/IlmBase/IlmBase.pc.in b/IlmBase/IlmBase.pc.in
index 1111111..2222222 100644
--- a/IlmBase/IlmBase.pc.in
+++ b/IlmBase/IlmBase.pc.in
@@ -13,5 +13,5 @@ Description: Base math and exception libraries
 Version: @ILMBASE_VERSION@
 Requires:
 Conflicts:
-Libs: -L${libdir} -lImath${libsuffix} -lHalf${libsuffix} -lIex${libsuffix} -lIexMath${libsuffix} -lIlmThread${libsuffix} @PTHREAD_LIBS@
+Libs.private: -L${libdir} -lImath${libsuffix} -lHalf${libsuffix} -lIex${libsuffix} -lIexMath${libsuffix} -lIlmThread${libsuffix} @PTHREAD_LIBS@
 Cflags: @PTHREAD_CFLAGS@ -I${includedir} -I${includedir}/OpenEXR
